testing/tests/DevExpress.ui.widgets.scheduler/integration.allDayAppointments.tests.js

```diff
@@ -1539,3 +1539,31 @@ QUnit.test("AllDay appointments should have correct position, groupOrientation =
     assert.roughEqual(firstPosition.top, firstAllDayRowPosition.top, 1.5, "Appointment has correct top");
     assert.roughEqual(secondPosition.top, secondAllDayRowPosition.top, 1.5, "Appointment has correct top");
 });
+
+QUnit.test("allDayPanel cell with custom dataCellTemplate must open appointment popup when double-clicked", function(assert) {
+    this.createInstance({
+        currentDate: new Date(2015, 4, 25),
+        views: ["week"],
+        currentView: "week",
+        firstDayOfWeek: 1,
+        startDayHour: 8,
+        endDayHour: 18,
+        height: 600,
+        dataCellTemplate: function(cellData, index, container) {
+            var wrapper = $("<div>").appendTo(container).addClass('dx-template-wrapper');
+            wrapper.append($("<div>").text(cellData.text).addClass("day-cell"));
+        }
+    });
+
+    this.instance.option("dataSource", [
+        { text: '1', startDate: new Date(2015, 4, 25), endDate: new Date(2015, 4, 26), allDay: true },
+        { text: '2', startDate: new Date(2015, 4, 25), endDate: new Date(2015, 4, 26), allDay: true },
+    ]);
+
+    var spy = sinon.spy(this.instance, "showAppointmentPopup");
+
+    var $allDayAppointment = $(this.instance.$element()).find(".dx-scheduler-all-day-appointment").eq(0);
```

Comment:
When @andrewmakarov see the similar code in the test he gets very angry. Use test wrapper!